bootstrap

```diff
@@ -4,6 +4,15 @@
 
 
 main(_Args) ->
+    case crypto:start() of
+        ok -> ok;
+        {error,{already_started,crypto}} -> ok
+    end,
```

Comment:
I'd switch to `application:start(crypto)` here instead. The reasoning is that the crypto app *used* to have a process to start but hasn't needed one since switching to NIF libraries. The function here is there as some sort of legacy and/or to load configs, but if it were ever to go or change what it does, `application:start/1` would keep working fine as it would then just load the lib and whatnot.